l10n_nl_tax_statement/models/l10n_nl_vat_statement.py

```diff
@@ -198,14 +198,30 @@ def _compute_amount_format_btw_total(self):
             btw = formatLang(self.env, statement.btw_total, monetary=True)
             statement.format_btw_total = btw
 
+    @api.model
+    def _get_unreported_move_from_date(self, from_date):
+        """ By default, the unreported date is not set before the start date
+        of the earliest report.
+        :param from_date: the from_date of the statement to compute the
+        unreported date for.
+        """
+        earliest = self.search([], order='from_date asc', limit=1).from_date
+        if not earliest:
+            return from_date
+        offset_date = from_date + relativedelta(months=-3, day=1)
```

Comment:
where do you get the three months from? My own company's vat duties are quarterly too, but that's not a given, right? Should we either derive the period from existing tax statements, or add a config parameter for the length?

Reply:
ah, the code below does that already, ignore then